src/libs/ReportActionsUtils.js

```diff
@@ -76,7 +97,55 @@ function isConsecutiveActionMadeByPreviousActor(reportActions, actionIndex) {
     return currentAction.action.actorEmail === previousAction.action.actorEmail;
 }
 
+/**
+ * Get the message text for the last action that was not deleted
+ * @param {Number} reportID
+ * @param {Object} [actionsToMerge]
+ * @return {String}
+ */
+function getLastVisibleMessageText(reportID, actionsToMerge = {}) {
+    const parser = new ExpensiMark();
+    const actions = _.toArray(lodashMerge({}, allReportActions[reportID], actionsToMerge));
+    const sortedActions = _.sortBy(actions, 'sequenceNumber');
+    const lastMessageIndex = _.findLastIndex(sortedActions, action => (
```

Comment:
Would it ever be possible for this to be undefined? Just wondering if we should maybe do an early return and skip the parsing. But even if it is possible, it probably happens super rarely.